packages/dds/map/src/mapKernel.ts

```diff
@@ -459,7 +403,7 @@ export class MapKernel {
         return true;
     }
 
-    public tryApplyStashedOp(op: any): unknown {
+    public tryApplyStashedOp(op: any): MapLocalOpMetadata {
```

Comment:
Alternative approach you might consider (to avoid public exposure of the metadata types) -- use the typings internally, but leave the map itself as returning `unknown` metadata, and have the `TestSharedMap` return the known typing.  This should avoid needing to export metadata types from the package boundary, which is perhaps desirable.